### ISSUE TITLE
Missing LICENSE file, and license mismatch between JSON metadata and src/*.js.

### ISSUE THREAD

**Issue by floatingatoll** [mozilla/promise-sqlite#2]
This repo is missing a LICENSE file; and, the package.json file says MPL-2.0; and, src/*.js opens with an Apache License header.

**Comment by rnewman** [comment]
Yup! Interested to hear what @Mossop wanted this licensed as in the end…

**Comment by Mossop** [comment]
I've pinged legal again to try to find out what we want here.
